lib/resque/scheduler.rb

```diff
@@ -92,7 +92,7 @@ def load_schedule!
         Resque.schedule.each do |name, config|
           load_schedule_job(name, config)
         end
-        Resque.redis.del(:schedules_changed)
+        Resque.redis.del(Resque.keys[:schedules_changed])
```

Comment:
I was referring to https://github.com/resque/resque/blob/v1.26.0/lib/resque.rb#L496-L502